client/javascripts/Transport.elm

```diff
@@ -0,0 +1,35 @@
+module Transport exposing (..)
+
+import Html exposing (..)
+
+-- MAIN
+
+main : Html String
```

Comment:
Extracting `view` from `main` without going straight to [TEA](https://guide.elm-lang.org/architecture/) strikes me as a somewhat odd choice; I'd be interested to know your reasoning. I've tended to start with `Html.beginnerProgram` as the next step up from a bare `main`.

Reply:
Yep, I'm prototyping. It'll certainly evolve into `Ports`, `Pages` and `Models` or `Data` for the type aliases.